pkg/platform/kube/project/external/leader/mlrun/client.go

```diff
@@ -0,0 +1,38 @@
+package mlrun
+
+import (
+	"github.com/nuclio/nuclio/pkg/platform"
+	"github.com/nuclio/nuclio/pkg/platform/kube/project/external/leader"
+	"github.com/nuclio/nuclio/pkg/platformconfig"
+
+	"github.com/nuclio/logger"
+	"github.com/nuclio/nuclio-sdk-go"
+)
+
+type Client struct {
+	leader.Client
```

Comment:
no need to explicit interface inheritance (see comment above)

Reply:
see my comment on the comment above 🙇‍♂️  (I want to keep it for the validations)

Reply:
agreed to make it implicit